CMake/psp_defs.cmake

```diff
@@ -0,0 +1,34 @@
+#General compilation options
+set(NONET ON)
+set(USE_SDL1 ON)
+set(NOSOUND ON)
+
+# keeping the 3DS defaults for now
+# Streaming audio is broken on the 3DS as of 25 Mar 2021:
+# https://github.com/devkitPro/SDL/issues/72
+set(DISABLE_STREAMING_MUSIC ON)

```

Comment:
This is no longer needed, it was due to a bug that has since been resolved.
PSP does not have enough RAM to run non-streaming audio